src/Data/Vec/Membership/Propositional/Properties.agda

```diff
@@ -13,7 +13,7 @@ open import Data.Product as Prod using (_,_; ∃; _×_; -,_)
 open import Data.Vec hiding (here; there)
 open import Data.Vec.Relation.Unary.Any using (here; there)
 open import Data.List.Base using ([]; _∷_)
-open import Data.List.Relation.Unary.Any using (here; there)
+open import Data.List.Relation.Unary.Any using (here; there) renaming (index to indexₗ)
```

Comment:
Could we use module imports rather than renaming? i.e. `open import Data.List.Relation.Unary.Any as ListAny using (here; there)` instead?